R/readIEA2024.R

```diff
@@ -0,0 +1,53 @@
+#' readIEA2024
+#'
+#' Read in energy balances from International Energy Agency until period 2022.
+#'
+#' @param subtype Type of data that should be read.
+#' @return The read-in data into a magpie object.
+#'
+#' @author Anastasis Giannousakis, Fotis Sioutas
+#'
+#' @examples
+#' \dontrun{
+#' a <- readSource("IEA2024", subtype = "MAINELEC")
```

Comment:
```
> str(a)
A magpie object (package: magclass)
 @ .Data:  num [1:1066716] NA NA NA NA NA NA NA NA NA NA ...
 $ dimnames:List of 3
  ..$ region                                   : chr [1:249] "ABW" "AFG" "AGO" "AIA" ...
  ..$ period                                   : chr [1:63] "y1960" "y1961" "y1962" "y1963" ...
  ..$ model.scenario.variable.unit.product.flow: chr [1:68] "(Missing).(Missing).(Missing).ktoe.HARDCOAL.MAINELEC" "(Missing).(Missing).(Missing).ktoe.BROWN.MAINELEC" "(Missing).(Missing).(Missing).ktoe.ANTCOAL.MAINELEC" "(Missing).(Missing).(Missing).ktoe.COKCOAL.MAINELEC" ...
```
the object returned has obsolete columns, maybe remove them?

Reply:
ok